### DISCUSSION TITLE
can we setup a public IPv6 with wireguard?

### DISCUSSION THREAD

**Discussion by benoitc** [pivpn/pivpn#1649] [Support]
By default IPv6 is nated with wireguard. Is there a way to use a full range /64 or /56 if needed  and allocate such a /64 or /128 to the users ?

**Comment by jbreitbart** [comment]
I was wondering the same thing. Have you managed to do this @benoitc?

**Comment by andreagdipaolo** [comment]
That would be nice. I hope someone introduces this feature.
